src/bin/solang.rs

```diff
@@ -434,6 +448,8 @@ fn compile(matches: &ArgMatches) {
         opt_level,
         log_api_return_codes,
         log_runtime_errors,
+        log_prints,
+        release,
```

Comment:
Here we craft an `Options` struct that can have invalid combinations (e.g. log prints enabled but also the release flag set). It is then up to anyone consuming the options struct later to always check for the release Flag. I don't think this is ideal. It would be better to get rid of this release flag and set the options accordingly.

```rust
log_prints &= !release;
log_api_return_codes &= !release;
log_runtime_errors &= !release;
```